src/ui/components/ListaProductos/filter/BpCheckedIcon.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from 'react';
+import { styled } from '@mui/material/styles';
+import Checkbox, { CheckboxProps } from '@mui/material/Checkbox';
+
+const BpIcon = styled('span')(({ theme }) => ({
+  borderRadius: 6,
+  width: 18,
+  height: 18,
+  border: '2px solid #CED4DA',
+  backgroundColor: theme.palette.primary.main,  
```

Comment:
Nice catch, this is the way to include the color, feel free to add the colors if you need inside the color palette.